### PR TITLE
Properly escape URL parameters when requesting `/test-executions`

### PR DESCRIPTION
**Pull request by 0x6675636b796f75676974687562**

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt [hunk 1]
```diff
@@ -405,7 +406,12 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
             filters = state.filters
             getData = { page, size ->
                 post(
-                    url = "$apiUrl/test-executions?executionId=${props.executionId}&page=$page&size=$size&checkDebugInfo=true",
+                    url = "$apiUrl/test-executions?" + URLSearchParams(jso {
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt:409]: Do we need to do the same with all other requests? Or why did you change only this one?

**Review by kgevorkyan — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt:409]: Could you create issue on this subject then? I also like this suggestion

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt:409]: If it's more safe/proper, can we move request params to `com.saveourtool.save.frontend.utils.RequestUtilsKt#post()` as map?

Comment (0x6675636b796f75676974687562) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt]: @nulls, please take a look -- I believe your suggestion is addressed with 647a52620d8d4da194a6e13194e83e2c37a534d7.

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt (hunk 1 above)
Comment (0x6675636b796f75676974687562) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt]: > Could you create issue on this subject then? I also like this suggestion

@kgevorkyan, see #1716.
